README.md

```diff
@@ -4,798 +4,3963 @@
+1. Run the following commands:
+   1. ```
+      $ git clone https://github.com/ODIM-Project/ODIM.git
+      ```
+   2. ```
+      $ cd ODIM
+      ```
+   3. ```
+      $ export ODIMRA_USER_ID=2021
+	   ```
+     ```
+   4. ```
+	   $ export ODIMRA_GROUP_ID=2021
+     ```
+   5. ```
+      $ ./build_images.sh
+      ```
+   6. ```
+      $ sudo docker images
+	   ```
+	    If the images are built successfully, you will receive an output which is similar to the following sample:
+	  ```
+     REPOSITORY                                TAG                                      IMAGE ID                   CREATED                     SIZE
+      consul                                    1.6                                       33ff2311df24               4 hours ago                  185MB
+      odim_zookeeper                            1.0                                      981d43f6c8b4              22 hours ago                 278MB
+      update                                    1.0                                      2cfb65430181              22 hours ago                 128MB
+      task                                      1.0                                      c4dd52b9ade0              22 hours ago                129MB
+      systems                                   1.0                                       9d3ad9845b16             22 hours ago                129MB
+      redis                                     1.0                                      81bf552d3a52              22 hours ago                 99.2MB
+      managers                                  1.0                                      2f7955586c54              22 hours ago                 128MB
+      odim_kafka                                1.0                                     f03a52363483              22 hours ago                  278MB
+      grf-plugin                                1.0                                     c7a086d02b16              22 hours ago                  100MB
+      fabrics                                   1.0                                     9b9ea8bafc30                22 hours ago                  128MB
+      events                                    1.0                                     860a9202a483              22 hours ago                  130MB
+      api                                       1.0                                     effab530ede5                22 hours ago                  130MB
+      aggregation                               1.0                                     354f67a857b6               22 hours ago                  130MB
+	   account-session                           1.0                                     a7eb07e69395              22 hours ago                  129MB
+	  ```
+2. Save each Docker image to a tar archive using the following command:
+    ```
+    $ sudo docker save -o <image_name.tar> <image_name:tag>
+    ```
+	Example: `sudo docker save -o consul.tar consul:1.0`
```

Comment:
Need to add a table with the list of ODIMRA images and corresponding archive name to be used for saving the images.